bot/src/slack.js

```diff
@@ -23,13 +23,12 @@ const addMessage = async (event) => {
 
   await upsertReport(report)
 
+  const isCommand = await handleCommands(event, web)
```

Comment:
I went back to returning boolean, because now it doesn't change the program flow, and it makes the function of `handleCommands` cleaner. It just looks for executable commands, and returns if it found any.